lib/ansible/modules/add_host.py

```diff
@@ -30,26 +30,21 @@
     type: list
     aliases: [ group, groupname ]
 extends_documentation_fragment:
-- action_common_attributes
+  - action_common_attributes
+  - action_core
 attributes:
     action:
         support: full
-    async:
-        support: none
-    become:
-        support: none
+    core:
+      description: While parts of this action are implemented in core, other parts are still available as normal plugins and can be partially overriden
```

Comment:
I don't think it is a good idea to use `description` anywhere outside the docs fragment. Having different descriptions for the same attribute in different plugins is very confusing.

Maybe it makes more sense to have a `comment` field where additional plugin-specific information can be added?

CC @samccann @abadger

Reply:
I think this depends a bit. I lost track but if an attribute can have 'partial' as it's support state, that means 'something' should describe what is partial about it.  I agree that ideally they all have the same description, but I don't know how many partials there might be out there. If there is only a few, then content owners could cover that in a note like they do today, or we could add another field for 'comments', but I feel like that would be a mostly empty field and take up space on the output?

Reply:
I think we miss a common property "idempotency", and especially for that something like "comments" is really important, since there are many modules that aren't 100% idempotent, and it is good to have a common place where the differences are documented.

Reply:
i was thinking of a notation to 'extend' instead of overwrite fragments for the case of description, i'm fine with adding comments but previously there was opposition about expanding the fields.

Reply:
Such an 'extend' mechanism would be useful for docs fragments in general, but I think in this case a different field would be a lot better, since the main description and module-specific comments on what the support value really means are two different pieces of informations which IMO shouldn't be mixed up.